packages/components/src/Accordion/AccordionHeader.tsx

```diff
@@ -9,21 +9,31 @@ export interface AccordionHeaderProps {
 }
 
 export const AccordionHeader = ({ children }: AccordionHeaderProps) => {
-  const { classNames } = useAccordionContext();
+  const { classNames, isStickyHeader, iconPosition } = useAccordionContext();
   /**
    * Use context to rotate the chevron.
    * "group-aria-expaned" is currently bugged with the RAC tailwind plugin.
    */
   const { isExpanded } = useContext(DisclosureStateContext)!;
+  const chevronStyles = cn(classNames.icon, isExpanded && 'rotate-180');
 
   return (
-    <Heading>
-      <Button slot="trigger" className={classNames.header}>
-        <div className="flex-1">{children}</div>
-        <ChevronDown
-          className={cn(classNames.icon, isExpanded && 'rotate-180')}
-        />
-      </Button>
-    </Heading>
+    <div
+      className={cn(
```

Comment:
does not need the `cn` function, I think. If you return `undefined` the `className` prop will not appear on the rendered HTML element